msmbuilder/featurizer/featurizer.py

```diff
@@ -347,6 +348,99 @@ def partial_transform(self, traj):
         return np.hstack(x)
 
 
+
+class AlphaAngleFeaturizer(Featurizer):
+    """Featurizer to extract alpha (dihedral) angles.
+
+    The alpha angle of residue `i` is the dihedral formed by the four CA atoms
+    of residues `i-1`, `i`, `i+1` and `i+2`.
+
+    Parameters
+    ----------
+    sincos : bool
+        Instead of outputting the angle, return the sine and cosine of the
+        angle as separate features.
+    """
+
+    def __init__(self, sincos=True):
+        self.sincos = sincos
+        self.atom_indices = None
+
+    def partial_transform(self, traj):
+
```

Comment:
do i change it in my thing and recommit and stuff or is there an easier way

On Tue, Nov 10, 2015 at 6:29 PM, Matthew Harrigan <notifications@github.com>
wrote:

> In msmbuilder/featurizer/featurizer.py
> <https://github.com/msmbuilder/msmbuilder/pull/691#discussion_r44495711>:
>
> > +    The alpha angle of residue `i` is the dihedral formed by the four CA atoms
> > +    of residues `i-1`, `i`, `i+1` and `i+2`.
> > +
> > +    Parameters
> > +    ----------
> > +    sincos : bool
> > +        Instead of outputting the angle, return the sine and cosine of the
> > +        angle as separate features.
> > +    """
> > +
> > +    def __init__(self, sincos=True):
> > +        self.sincos = sincos
> > +        self.atom_indices = None
> > +
> > +    def partial_transform(self, traj):
> > +
>
> remove extra empty line
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/msmbuilder/msmbuilder/pull/691/files#r44495711>.
>



-- 

Brooke Husic
PhD student in chemistry
stanford | bhusic@stanford.edu
cell phone | +1 860 299 5434